### ISSUE TITLE
OSError: dlopen(.../lib/python3.8/site-packages/matx/lib/libmatx.dylib, 0x000A): Library not loaded: /Users/bytedance/Desktop/maxiandi/github/matxscript/ci/../lib/pcre/lib/libpcre.1.dylib

### ISSUE THREAD

**Issue by ChandlerBent** [bytedance/matxscript#135]
version: v1.8.0
OS: mac OS 12.3.1 (21E258)

```
OSError: dlopen(.../lib/python3.8/site-packages/matx/lib/libmatx.dylib, 0x000A): Library not loaded: /Users/bytedance/Desktop/maxiandi/github/matxscript/ci/../lib/pcre/lib/libpcre.1.dylib
  Referenced from: .../lib/python3.8/site-packages/matx/lib/libmatx.dylib
  Reason: tried: '/Users/bytedance/Desktop/maxiandi/github/matxscript/ci/../lib/pcre/lib/libpcre.1.dylib' (no such file), '/usr/lib/libpcre.1.dylib' (no such file)
```

The `libpcre.1.dylib` file is exists `lib` folder of `matx` package
It seems still read develop environment file path.

**Comment by vermouth1992** [comment]
Is it macOS m1 or macOS x64?

**Comment by cloud-mxd** [comment]
Thanks for your feedback! We will fix this as soon as possible.
Until it's fixed, maybe you can bypass via `brew install pcre`

**Comment by cloud-mxd** [comment]
Fixed for x86_64, you can try
`pip install matxscript --no-cache-dir --force-reinstall`

**Comment by cloud-mxd** [comment]
Fixed in 1.8.1
